packages/unified-pipeline/src/components/Canvas/utils/LROrientation/Constants.ts

```diff
@@ -7,7 +7,7 @@ export const NODE_DEFAULT_HEIGHT = 50
 export const NODE_HORIZONTAL_MARGIN = 25
 /* Increase margin size to avoid potential overalap with node labels */
 export const NODE_HORIZONTAL_MARGIN_2 = 2 * NODE_HORIZONTAL_MARGIN
-export const NODE_VERTICAL_MARGIN = 25
+export const NODE_VERTICAL_MARGIN = 60
```

Comment:
This is causing lot of white space. Let's discuss how we can improve this.
<img width="958" alt="image" src="https://github.com/user-attachments/assets/1d43e53f-9397-4b9b-8a9e-de965bab7daf">